cluster-controller/src/main/java/io/strimzi/controller/cluster/resources/AbstractCluster.java

```diff
@@ -21,6 +21,8 @@
 
     protected final Logger log = LoggerFactory.getLogger(getClass());
 
+    public static final String STRIMZI_CLUSTER_CONTROLLER_DOMAIN = "cluster-contorller.strimzi,io";
```

Comment:
Unfortunately, you copied it from my comment with the `,` instead of `.` - Sorry, that was my typo :-(.

Reply:
I remember your comment was worse (a typo in "strimzi") :P but I didn't notice all the other typos even on "contorller" :)

Reply:
yeah, I see that it had maaany typos :-o